#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_adjust_docbookxsl_path.dpatch by Daniel Leidert (dale) <daniel.leidert@wgdd.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: We will use docbook(-xsl) 4.3 distributed by Debian GNU/Linux.

@DPATCH@
diff -urNad bluefish-doc-0.0.20050201/Makefile /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/Makefile
--- bluefish-doc-0.0.20050201/Makefile	2005-02-01 16:28:41.000000000 +0100
+++ /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/Makefile	2005-02-01 20:24:07.000000000 +0100
@@ -30,7 +30,7 @@
 BOOK_PS_US_TARGET=${OUTPUT_DIR}/book-us.ps
 BOOK_XML_SOURCE=${BOOK_DIR}/book.xml
 
-XSL_DIR=http://docbook.sourceforge.net/release/xsl/current
+XSL_DIR=/usr/share/xml/docbook/stylesheet/nwalsh
 
 HAVE_XSL_DIR=$(shell test -d docbook-xsl && echo "0")
 ifeq (${HAVE_XSL_DIR},0)
diff -urNad bluefish-doc-0.0.20050201/stylesheets/db2html.xsl /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/db2html.xsl
--- bluefish-doc-0.0.20050201/stylesheets/db2html.xsl	2005-02-01 16:28:41.000000000 +0100
+++ /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/db2html.xsl	2005-02-01 20:25:42.000000000 +0100
@@ -3,7 +3,7 @@
                 version='1.0'
                 xmlns="http://www.w3.org/TR/xhtml1/transitional"
                 exclude-result-prefixes="#default">
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl"/>
+<xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/html/chunk.xsl"/>
 <!-- <xsl:import href="../xsl/html/chunk.xsl"/> -->
 <xsl:import href="titlepage-html.xsl"/>
 <!-- Allow to use extensions -->
diff -urNad bluefish-doc-0.0.20050201/stylesheets/db2longhtml.xsl /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/db2longhtml.xsl
--- bluefish-doc-0.0.20050201/stylesheets/db2longhtml.xsl	2005-02-01 16:28:41.000000000 +0100
+++ /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/db2longhtml.xsl	2005-02-01 20:25:55.000000000 +0100
@@ -3,7 +3,7 @@
 	version='1.0' 
 	xmlns="http://www.w3.org/TR/xhtml1/transitional" 
 	exclude-result-prefixes="#default">
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/docbook.xsl"/>
+<xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/html/docbook.xsl"/>
 <!-- <xsl:import href="../xsl/html/docbook.xsl"/> -->
 <xsl:import href="titlepage-html.xsl"/>
 <!-- Allow to use extensions -->
diff -urNad bluefish-doc-0.0.20050201/stylesheets/pdf-ps-a4.xsl /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/pdf-ps-a4.xsl
--- bluefish-doc-0.0.20050201/stylesheets/pdf-ps-a4.xsl	2005-02-01 16:28:41.000000000 +0100
+++ /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/pdf-ps-a4.xsl	2005-02-01 20:26:13.000000000 +0100
@@ -2,7 +2,7 @@
 	xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
 	xmlns:fo="http://www.w3.org/1999/XSL/Format">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/fo/docbook.xsl"/>
+<xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/fo/docbook.xsl"/>
 <!-- <xsl:import href="../xsl/fo/docbook.xsl"/> -->
 <xsl:import href="titlepage-a4.xsl"/>
 
diff -urNad bluefish-doc-0.0.20050201/stylesheets/pdf-ps-us.xsl /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/pdf-ps-us.xsl
--- bluefish-doc-0.0.20050201/stylesheets/pdf-ps-us.xsl	2005-02-01 16:28:41.000000000 +0100
+++ /tmp/dpep.LEJptD/bluefish-doc-0.0.20050201/stylesheets/pdf-ps-us.xsl	2005-02-01 20:26:27.000000000 +0100
@@ -2,7 +2,7 @@
 	xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
 	xmlns:fo="http://www.w3.org/1999/XSL/Format">
 
-<xsl:import href="http://docbook.sourceforge.net/release/xsl/current/fo/docbook.xsl"/>
+<xsl:import href="/usr/share/xml/docbook/stylesheet/nwalsh/fo/docbook.xsl"/>
 <!-- <xsl:import href="../xsl/fo/docbook.xsl"/> -->
 <xsl:import href="titlepage-us.xsl"/>
 
